tests/h/services/annotation_stats_test.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+import pytest
+
+from h.services.annotation_stats import AnnotationStatsService
+from h.services.annotation_stats import annotation_stats_factory
+
+
+class TestAnnotationStatsService(object):
+    @pytest.mark.usefixtures('user')
+    def test_user_annotation_counts_returns_count_of_annotations_for_user(self, svc, user, factories, db_session):
```

Comment:
Actually the same thing is true here. You don't need the user fixture at all. Just make up a userid, much as you've done for the group pubid.